src/ServiceContainer/SymfonyExtension.php

```diff
@@ -63,6 +78,8 @@ public function configure(ArrayNodeDefinition $builder): void
                         ->scalarNode('class')->defaultNull()->end()
                         ->scalarNode('environment')->defaultNull()->end()
                         ->booleanNode('debug')->defaultNull()->end()
+                        ->booleanNode('autoconfigure')->defaultFalse()->end()
```

Comment:
I guess those options are not really related to the kernel configuration, so it should be on the same level with `bootstrap` and `kernel` options.